randovania/data/json_data/prime2.txt

```diff
@@ -2341,20 +2490,23 @@ Asset id: 3619928121
           Morph Ball and Spider Ball
           Any of the following:
               Dark Beam and Light Beam and Light Suit and Morph Ball Bomb
-              Dark Beam and Light Beam and Morph Ball Bomb and Dark World Damage ≥ 800 and Difficulty: Normal
-              Dark Beam and Light Beam and Power Bomb ≥ 3 and Dark World Damage ≥ 800 and Difficulty: Normal
-              Annihilator Beam and Morph Ball Bomb and Light Ammo ≥ 150 and Dark World Damage ≥ 800 and Difficulty: Normal
-              Annihilator Beam and Power Bomb ≥ 3 and Light Ammo ≥ 150 and Dark World Damage ≥ 800 and Difficulty: Normal
-              Morph Ball Bomb and Space Jump Boots and Screw Attack and Dark World Damage ≥ 800 and Difficulty: Normal
-              Space Jump Boots and Screw Attack and Power Bomb ≥ 3 and Dark World Damage ≥ 800 and Difficulty: Normal
+              Dark Beam and Light Beam and Morph Ball Bomb and Movement (Normal) and Dark World Damage ≥ 800 and Difficulty: Normal
+              Dark Beam and Light Beam and Power Bomb ≥ 3 and Movement (Normal) and Dark World Damage ≥ 800 and Difficulty: Normal
+              Annihilator Beam and Morph Ball Bomb and Light Ammo ≥ 150 and Movement (Normal) and Dark World Damage ≥ 800 and Difficulty: Normal
+              Annihilator Beam and Power Bomb ≥ 3 and Light Ammo ≥ 150 and Movement (Normal) and Dark World Damage ≥ 800 and Difficulty: Normal
+              All of the following:
+                  Morph Ball Bomb and Movement (Normal) and Dark World Damage ≥ 800 and Difficulty: Normal
+                  Use Screw Attack (Space Jump)
+              All of the following:
+                  Power Bomb ≥ 3 and Movement (Normal) and Dark World Damage ≥ 800 and Difficulty: Normal
+                  Use Screw Attack (Space Jump)
 
 > Event - Emperor Ing; Heals? True
   > Sanctum/Door to Sanctum Access
       All of the following:
-          Screw Attack
-          Any of the following:
-              Space Jump Boots and Dark World Damage ≥ 40
-              Screw Attack without Space Jump (Normal) and Dark World Damage ≥ 50 and Difficulty: Normal
+          Screw Attack without Space Jump (Normal) and Dark World Damage ≥ 40 and Dark World Damage ≥ 50 and Difficulty: Normal
```

Comment:
This seems incorrect

Reply:
Oh I see the issue. Somehow it didn't group correctly.